usr/src/cmd/Makefile

```diff
@@ -346,6 +346,7 @@ COMMON_SUBDIRS=	\
 	setmnt		\
 	setpgrp		\
 	sgs		\
+	sh		\
```

Comment:
I wasn't going to bother, at least as a first pass, shipping the legacy `usr/has` stuff on ARM.

Reply:
Ok I drop this then,